test/e2e_node/e2e_node_suite_test.go

```diff
@@ -89,12 +94,13 @@ func registerNodeFlags(flags *flag.FlagSet) {
 }
 
 func init() {
-	// Enable bindata file lookup as fallback.
-	e2etestfiles.AddFileSource(e2etestfiles.BindataFileSource{
-		Asset:      generated.Asset,
-		AssetNames: generated.AssetNames,
-	})
-
+	// Enable embedded FS file lookup as fallback
+	// TODO: This needs to be cleaned up later to include only file sources which are necessary.
+	e2etestfiles.AddFileSource(e2etestingmanifests.GetE2ETestingManifestsFS())
+	e2etestfiles.AddFileSource(e2enodetestingmanifests.GetE2ENodeTestingManifestsFS())
+	e2etestfiles.AddFileSource(testimages.GetTestImagesFS())
+	e2etestfiles.AddFileSource(testfixtures.GetTestFixturesFS())
+	e2etestfiles.AddFileSource(conformancetestdata.GetConformanceTestdataFS())
```

Comment:
@kubernetes/sig-node-pr-reviews -- does the e2e_node test suite require test fixtures from all the sources?

I noticed the usage of "test/e2e_node/testing-manifests", asking to be sure about all other test data usages and for backward compatibility.